models/contracts/optimism/contracts_optimism_nondeterministic_contract_creators.sql

```diff
@@ -0,0 +1,29 @@
+{{ 
+  config(
+    alias='nondeterministic_contract_creators',
+    unique_key='creator_address',
```

Comment:
this unique_key won't apply any tests on it's own. it's mainly used for incremental models to know which columns to join on for merge statement. since this is view, that doesn't apply here. the unique test needs applied in schema file on column name.

Reply:
Got it, the original instantiation of the column has a unique test on it, so I think it will come through (couldn't get dbt to compile when I added the tests to a *contract_creator column)